include/net/net_context.h

```diff
@@ -312,6 +312,9 @@ __net_socket struct net_context {
 #endif
```

Comment:
Please squash the commits together and change the commit message to
```
net: tcp: Add Keepalive support

Implement TCP keepalive support from RFC 1122

Fixes #34044

Signed-off-by: <your id etc>
```

Reply:
I don't know how to operate it, can you teach me the specific command line code

Reply:
There are multiple ways, but I usually do all tree manipulations with interactive git rebase.
So something like this:  `git rebase -i HEAD~2`, then mark the latter commit with "s" (which means squash), and mark the first commit with "r" (which means reword and edit the commit msg) and save.

Also, we do not do merge commits in Zephyr so you need to rebase properly. Please, do not close this PR but just force push a new version `git push -f origin <your-branch>`

Reply:
Is my last commit redundant? How to revoke it?